src/pages/Summary/index.js

```diff
@@ -1,11 +1,139 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { selectToken } from '../../store/user/selectors'
+import {
+	selectUserRecords,
+	selectCountOfUserRecords,
+} from '../../store/record/selectors'
+import { fetchAllUserRecords } from '../../store/record/actions'
+import Loading from '../../components/Loading'
+
+import './SummaryPage.css'
 
 const Summary = () => {
-  return (
-    <div>
-      Summary
-    </div>
-  )
+	const dispatch = useDispatch()
+	const history = useHistory()
+	const token = useSelector(selectToken)
+	const records = useSelector(selectUserRecords)
+	const count = useSelector(selectCountOfUserRecords)
+
+	useEffect(() => {
+		if (records.length === 0) {
+			dispatch(fetchAllUserRecords())
+		}
+	}, [])
+
+	if (!records || records.length === 0) {
+		return <Loading />
+	}
+
+	if (!token) {
+		history.push('/')
+	}
+	const sortedPrice = [...records].sort((a, b) => b.lowestPrice - a.lowestPrice)
+
+	const worthOne = sortedPrice.length >= 1 ? sortedPrice[0] : null
+	const worthTwo = sortedPrice.length >= 2 ? sortedPrice[1] : null
+	const worthThree = sortedPrice.length >= 3 ? sortedPrice[2] : null
```

Comment:
&& instead of ternary, rule of thumb, if one of the returns is null then you can fix it with && or ||